web/src/scripts/webflow/methods.mjs

```diff
@@ -15,15 +15,15 @@ const siteId = "5e31b06cb76b830809358a75";
 // eslint-disable-next-line no-undef
 const headers = { Authorization: `Bearer ${process.env.WEBFLOW_API_TOKEN}` };
 
-const getAllItems = async (id) => {
+const getAllItems = async (collectionId) => {
```

Comment:
I like this. It's a good case of changing a variable name to make it more clear!